com.microsoft.mrtk.input/Interactors/InteractorVisuals/MRTKLineVisual.cs

```diff
@@ -466,7 +461,16 @@ private bool TryFindLineRenderer()
             return true;
         }
 
-        [Range(2, 2048)]
+        private void ClearLineRenderer()
+        {
+            if (TryFindLineRenderer())
+            {
+                lineRenderer.SetPositions(clearPositions);
+                lineRenderer.positionCount = 0;
+            }
+        }
+
+        [Range(2, 128)]
```

Comment:
Are we intentionally changing the range here?

Reply:
yes, the previous range added lots of points to the line renderer which would all be updated in real time.